### PR TITLE
fix(ResourceLabel): mark resource as updated if labels changed

### PR DESCRIPTION
**Pull request by Lawrence-lkq**
### Description

Fixes # (issue)

### Checklist

- [ ] 填写 PR 描述及相关 issue (write PR description and related issue)
- [√] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [ ] 单元测试通过 (unit test passed)
- [√] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

File: src/dashboard/apigateway/apigateway/apis/web/resource/views.py [hunk 1]
```diff
@@ -351,6 +352,10 @@ def update(self, request, *args, **kwargs):
             label_ids=slz.validated_data["label_ids"],
         )
 
+        # 更新标签也要更新资源的更新时间
+        instance.updated_time = datetime.now()
+        instance.save()
```
Comment (wklken) [inline src/dashboard/apigateway/apigateway/apis/web/resource/views.py:357]: 直接调用instance.save即可， updated_time auto_now=True，会自动更新

即， 356 行可以去掉

File: src/dashboard/apigateway/apigateway/apis/web/resource/views.py [hunk 2]
```diff
@@ -351,6 +352,10 @@ def update(self, request, *args, **kwargs):
...
+
         return OKJsonResponse(status=status.HTTP_204_NO_CONTENT)
```
Comment (wklken) [inline src/dashboard/apigateway/apigateway/apis/web/resource/views.py:360]: 单测需要补一下， assert updated_time在调用方法后有变更

File: src/dashboard/apigateway/apigateway/apis/web/resource/views.py (hunk 1 above)
Comment (Lawrence-lkq) [reply src/dashboard/apigateway/apigateway/apis/web/resource/views.py:355]: 已更改

File: src/dashboard/apigateway/apigateway/apis/web/resource/views.py (hunk 2 above)
Comment (Lawrence-lkq) [reply src/dashboard/apigateway/apigateway/apis/web/resource/views.py:358]: 已添加单测并单测通过

**Review by wklken — APPROVED** [review]

**Pull request merged**